fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -187,6 +184,14 @@ def self.run(options)
         end
       end
 
+      def self.puts_download_url(body)
+        version_url = body['public_url']
+        version_url = body['public_url'] + '/app_versions/' + body['config_url'].split('/').last if body['config_url']
+        version_url = body['public_url'] + '/app_versions/' + response.body['version'] if body['version']
+        Actions.lane_context[SharedValues::HOCKEY_DOWNLOAD_LINK] = version_url
```

Comment:
Too focused on nil checks to notice it also sets this shared value. As the method name starts with `puts_` I would not expect side effects